lib/typhoeus/request.rb

```diff
@@ -214,7 +214,9 @@ def set_defaults
       options[:headers] = {'User-Agent' => default_user_agent}.merge(options[:headers] || {})
       options[:verbose] = Typhoeus::Config.verbose if options[:verbose].nil? && !Typhoeus::Config.verbose.nil?
       options[:maxredirs] ||= 50
+      @options[:headers]['Expect'] ||= ''
```

Comment:
1. Thanks @ramigluzman , though all the credit for this is to @linrock
2. True, fixed.
3. This condition creates the option hash if it's empty, with a pre defined User Agent value. it's not the same case. I want to assign a default value inside the existing headers hash regardless if it was never created or it was created, but with out the `Expect` option.